pyproject.toml

```diff
@@ -72,5 +72,32 @@ sympy = "==1.12"
 requires = ["setuptools","poetry>=0.12"]
 build-backend = "poetry.masonry.api"
 
+[tool.ruff]
+# exclude = ["__init__.py"]
+line-length = 120
+target-version = "py311"
 
+[tool.ruff.lint]
+select = ["I", "D", "E", "W"]
+# ruff configuration
+# E, W -- pycodestyle
+# D -- pydocstyle
+# I -- isort
```

Comment:
@vprusso Of the available rules to check, these appeared to be the most obvious. Do you want me to [anything else](https://docs.astral.sh/ruff/rules/#rules)?

BTW feel free to take your time on responding to things on here. I don't really expect a quick response on holidays and weekends.

Reply:
@vprusso You missed this earlier comment here. No worries!

Should we try to add more rules beyond what's already added?

Reply:
Ah, my apologies. Nope, I think that list looks good. If we decide we want to add something in later we can do that, but for now, this list looks good to me!